### ISSUE TITLE
Database navigator - see table size

### ISSUE THREAD

**Issue by miky-schreiber** [dbeaver/dbeaver#11538]
It would be great to have the table size in terms of number of rows inside the database navigator, similar to the table size in GB that we now have (see PR #8705) - 

![image](https://user-images.githubusercontent.com/61650261/109411204-0d081700-79a9-11eb-887f-cd493dbecfef.png)

We can show both table size and number of rows or make it configurable.

**Comment by uslss** [comment]
thanks for the suggestion

**Comment by fujiokae** [comment]
I completely agree! I have been using pgAdmin III for a long time and just recently started using DBeaver. 
With pgAdmin, you can right-click on a table and select "Count" to get the accurate number of rows in the rigtht-side panel. With DBeaver, you have to View Table, switch to [Properties] tab, go to "Statistics". Then, right-click and choose "Read row count...".

It would be great if the Database Navigator can display the number of rows (estimated first is OK but should provide "Count row" option in the right-click menu). Also, it seems there is a bug that does not display the table size when "Custom" view is selected (the table size is displayed when "Simple" view is selected).

DBeaver 21.0.2.202104042040

**Comment by Progress1** [comment]
Very useful. I use it in other tools but i miss it in DBeaver. If you are afraid of some performance on startup please add it to some setting (default OFF) so users can choose what they want.

**Comment by guttizin** [comment]
I was researching exactly **how to show the row count** (instead of table size), in the database navigator, and I came across this open issue.
Would be great to see this feature in development.

**Comment by superwinni2** [comment]
+1 for the feature!
